spring-webmvc/src/test/java/org/springframework/web/servlet/mvc/method/annotation/ServletRequestMethodArgumentResolverTests.java

```diff
@@ -44,6 +31,21 @@
 import org.springframework.web.testfixture.servlet.MockHttpServletResponse;
 import org.springframework.web.testfixture.servlet.MockHttpSession;
 
+import javax.servlet.ServletRequest;
```

Comment:
Please adhere to the [project guidelines for imports](https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements).

In addition, please execute `./gradlew check` before submitting a PR or pushing changes to a PR.

Reply:
Done, sorry about that